lazy_tensor_core/lazy_tensor_core/csrc/tensor_aten_ops.cpp

```diff
@@ -291,13 +291,6 @@ LazyTensor DispatchComparisonOp(c10::Symbol kind, const LazyTensor& input,
   return LazyTensor::Create(node, input.GetDevice(), at::ScalarType::Bool);
 }
 
-// Same as above, with the second input a tensor as well.
-LazyTensor DispatchComparisonOp(c10::Symbol kind, const LazyTensor& input,
```

Comment:
can we also delete DispatchComparisonOp? or still used by something?

Reply:
It's used by `lt.Scalar` which is then used by some index ops. So we might need to revisit this once all depended ops are cleaned-up/code-generated.